extensions/sql-database-projects/src/models/project.ts

```diff
@@ -567,6 +571,16 @@ class SystemDatabaseReferenceProjectEntry extends DatabaseReferenceProjectEntry
 	constructor(uri: Uri, public ssdtUri: Uri, public name: string) {
 		super(uri, DatabaseReferenceLocation.differentDatabaseSameServer, name);
 	}
+
+	public pathForSqlProj(): string {
+		// need to remove the leading slash for system database path for build to work on Windows
+		return utils.convertSlashesForSqlProj(this.fsUri.path.substring(1));
+	}
+
+	public ssdtPathForSqlProj(): string {
+		// need to remove the leading slash for system database path for build to work on Windows
+		return utils.convertSlashesForSqlProj(this.ssdtUri.path.substring(1));
```

Comment:
does this URI always have a leading slash, or does it need to be checked before it gets hit with `substring(1)`?

Reply:
yup the URI for the system databases always has a leading slash